src/components/m-annotator.vue

```diff
@@ -239,6 +249,16 @@ export default {
         img.style.setProperty('display', 'block')
 
         if (this.read_only && (this.initial_imdata.boxes === null || this.initial_imdata.boxes.length === 0)){
+            let paper = this.paper;      
+            paper.activate(); 
+            let cnv = this.$refs.canvas;
+            console.log('drawing canvas', img.height, img.width, img)
+            cnv.height = height;
+            cnv.width = width;
+            this.paper.setup(cnv);
+            this.height_ratio = height / img.naturalHeight
+            this.width_ratio = width / img.naturalWidth
+            this.paper.view.draw();
             return;
         }
         // call some code to draw activation array 
```

Comment:
seems like the code in the branch above is the same as outside the branch. can we avoid the duplication and just skip on the loading box data?